### PR TITLE
Added author support (gradle/guides#114)

### PR DESCRIPTION
**Pull request by ysb33r**
@bmuschko This adds support for showing the author. I have already listed you as the main author.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+    // id 'org.gradle.guides.test-jvm-code' version '0.5.3'
 }
 
-ext {
-    repoPath = "gradle-guides/implementing-gradle-plugins"
+import org.apache.tools.ant.filters.*
+
+guide {
+    repoPath 'gradle-guides/implementing-gradle-plugins'
+    mainAuthor 'Ben Muschko'
```
Comment (bmuschko) [inline build.gradle]: Maybe use my full name: "Benjamin".

File: build.gradle [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 }
 
-apply from : 'gradle/build-documentation.gradle'
-apply from : 'gradle/publish-documentation.gradle'
-apply from : 'gradle/test-source.gradle'
+// Add in any generation steps.
+gradleRunner {
```
Comment (bmuschko) [inline build.gradle]: Looks like a stray artifact.

Comment (ysb33r) [inline build.gradle]: Yes, you probably don't need it. It just comes in from the template. I'll remove it.

**Comment by ysb33r** [comment]
@bmuschko Done

**Pull request merged**